### ISSUE TITLE
basic usage code need to run twice to get it work

### ISSUE THREAD

**Issue by enzofxxjia** [brendan-w/python-OBD#32]
Hello Brendan 

I just want to test the basic usage code for my car. 
code is here 
```python
import obd

connection = obd.OBD("/dev/rfcomm0")
cmd = obd.commands.RPM # select an OBD command (sensor)

response = connection.query(cmd) # send the command, and parse the response

print(response.value)
print(response.unit)
obd.debug.console = True
```
every time I run this code, I need to run it twice 
The first time run, I got 
```python
[obd] '010E: RPM is not supported '
```
after that I close the program and run again 
It work prefect,I could read rpm in the python shell 
Any idea on how to fix that ?
Thanks

**Comment by brendanwhitfield** [comment]
Hi! Please move the `obd.debug.console = True` line to just after `import obd`. That way, it will print all of the debug data from the connection process (which is probably where the problem is).

**Comment by enzofxxjia** [comment]
Hi Brendan thanks for reply
I common out that code ```obd.debug.console = True```
But still need to run twice to get it print on the python shell. 
Do I need to put a delay between the 
```python 
response = connection.query(cmd) # send the command, and parse the response
```
and 
```python 
print(response.value)
```

**Comment by brendanwhitfield** [comment]
Nope, don't comment anything out.

Here, run this script, and send me all of the output please:

```python
import obd

# needs to be set BEFORE you make a connection
obd.debug.console = True

connection = obd.OBD("/dev/rfcomm0")
cmd = obd.commands.RPM # select an OBD command (sensor)

response = connection.query(cmd) # send the command, and parse the response

print(response.value)
print(response.unit)
```

**Comment by enzofxxjia** [comment]
This photo is the output of the first run 
![img_6788 1](https://cloud.githubusercontent.com/assets/15572354/10987678/a63a2720-8403-11e5-91bc-20a60db6ea36.JPG)
something error with the ATE0 command 
than, I close the program and run again 
It pass this error 
![img_6790](https://cloud.githubusercontent.com/assets/15572354/10987719/fb3c60bc-8403-11e5-9660-452f5c06a163.JPG)
![img_6791 1](https://cloud.githubusercontent.com/assets/15572354/10987718/fb3ab7ee-8403-11e5-9312-d305dde55913.JPG)

I am using the raspberry PI put run this code 
However,if I use the my laptop (win10), I didn't get any error in the first run.

**Comment by brendanwhitfield** [comment]
Are you using code from `devel`? Version 0.4.1 doesn't send `ATSP0`. If you are, word of warning: there may be a lot of things broken on `devel`. I only recently got basic functionality working.

Anyhow, it looks like the initial reset command didn't get transmitted properly (and caused one of the following commands to fail). If it works on your laptop, then it's probably an eccentricity with the Pi's bluetooth connection. It's not a big deal. I've made a fix on `devel` that will be released with 0.5.0. If you're not using devel, or if you cant wait, here's the diff that should fix the problem c091439.
